kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/cluster/Kubernetes.java

```diff
@@ -54,4 +54,14 @@ public KubeCmdClient defaultCmdClient() {
     public String toString() {
         return CMD;
     }
+
+    public boolean isOpenshift() {
+        try {
+            return Exec.exec(CMD, "api-versions").out().contains("openshift.io");
+        }
+        catch (KubeClusterException e) {
+            LOGGER.debug(e.getMessage());
```

Comment:
Lots of bad practice here.

* what if `e.getMessage()` contains a `{}`?  the program will fail at runtime with a missing variable exception, which will mislead a person investigating by obscuring root case/waste time.  The formatted message passed to a logging library needs to be under your control - **always**.   So..  `LOGGER.debug("{}", e.getMessage());` is better.. but....
* this is an error condition, so we should treat it that way.   
 ** log at error
 ** use a meaningful message. "Failed whilst sniffing for OpenShift" and chain the exception.
 ** don't swallow the error (i.e. don't return false).